<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>themesOptions</class>
 <widget class="QWidget" name="themesOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>389</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>296</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Use</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" colspan="2">
        <widget class="QCheckBox" name="alwaysOnTop"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_startMinimized">
         <property name="text">
          <string>Start minimized</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="startMinimized">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Minimize on close to systray</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="minimizeToSystray"/>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_copyEnd">
         <item>
          <property name="text">
           <string notr="true">Don't close if errors are found</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Never close</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Always close</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>At the end of the copy</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_Slider_speed">
         <property name="text">
          <string>Limit copy speed to</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="SliderSpeed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SpeedMaxValue">
           <property name="text">
            <string notr="true">Unlimited</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="checkBox_limitSpeed">
         <property name="text">
          <string>Limit copy speed to</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QSpinBox" name="limitSpeed">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> KB/s</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>133</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelStartWithMoreButtonPushed">
         <property name="text">
          <string>Start with the &quot;more button&quot; pushed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxStartWithMoreButtonPushed"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelAlwaysOnTop">
         <property name="text">
          <string>Always on top</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelSavePosition">
         <property name="text">
          <string>Save position</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="savePosition">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>296</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxShowSpeed"/>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>529</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelShowSpeedAsMain">
         <property name="text">
          <string>Show speed as main information</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="fileProgression">
         <item>
          <property name="text">
           <string>Done files</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In progress files</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>File progression</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>296</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Display</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QFrame" name="frameProgressionColor">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to edit the color</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="progressColorWrite">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="iconSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="progressColorRead">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="iconSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="progressColorRemaining">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="iconSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelSpeedWithProgressBar">
         <property name="text">
          <string>Speed with progress bar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="showProgressionInTheTitle"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelProgressionColor">
         <property name="text">
          <string>Progression color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="showDualProgression"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Show progression in the title</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>General margin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDualProgression">
         <property name="text">
          <string>Show dual progression</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>455</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="speedWithProgressBar"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>General spacing</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="generalMargin"/>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="generalSpacing"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_limitSpeed</sender>
   <signal>clicked(bool)</signal>
   <receiver>limitSpeed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
